### PR TITLE
issue 2: started code by adding plugin template files

### PR DESCRIPTION
**Pull request by nipendar**
tested adding it to formean Gemfile, no errors

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkg/
+test/dummy/db/*.sqlite3
+test/dummy/log/*.log
+test/dummy/tmp/
+test/dummy/.sass-cache
+locale/*.mo
+locale/*/*.edit.po
+locale/*/*.po.time_stamp
+locale/*/*.pox
+.rstrip
```
Comment (edestecd) [inline .gitignore]: Wanna add some of theses?
https://github.com/MiamiOH/foreman_custom_tab/blob/master/.gitignore

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.4.2
```
Comment (edestecd) [inline .ruby-version]: Don't check in this file: put it in .gitignore

File: config/routes.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
```
Comment (edestecd) [inline config/routes.rb]: lets leave this file out until we actually need it

File: app/controllers/concerns/foreman_teamdynamix/hosts_controller_extensions.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module ForemanTeamdynamix
```
Comment (edestecd) [inline app/controllers/concerns/foreman_teamdynamix/hosts_controller_extensions.rb]: Lets leave out all the example files in app until they have real and useful code in them.

**Comment by edestecd** [comment]
Were missing .tx and Rakefile

**Comment by nipendar** [comment]
Done

**Pull request merged**